assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -239,6 +239,7 @@ export default createHigherOrderComponent(
 				opacity,
 				type: textBlockTextType,
 				ampShowImageCaption,
+				ampShowCaption,
```

Comment:
I was originally thinking of using just `ampShowCapion` for both of these instead of separating since the functionality is really the same for both attributes, however, since we released already then the previously saved `ampShowImageCaption` would get lost when changing the attribute's name.

@swissspidy Any thoughts if there's a better way to do it without losing the previous attribute while not duplicating the logic for just a different attribute name?

Reply:
Hmm normally this would be a use case for [deprecated blocks](https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/), but since this extends existing blocks, I don't have an optimal solution off the top of my head.

If it makes things easier you could probably do something like this:

```js
const captionAttribute = isVideoBlock ? 'ampShowCaption' : 'ampShowImageCaption;
// ...
checked={ attributes[ captionAttribute ] }
// ...
props.setAttributes( { [ captionAttribute ]: ! attributes[ captionAttribute ] } );
```

Reply:
Changed, thanks!